### PR TITLE
Pdi 25 19 18 86 32 33 31 resolve conflicts

### PR DESCRIPTION
**Pull request by CarlHA**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature [hunk 1]
```diff
@@ -5,8 +5,8 @@ Feature: Non-levy learner 19-24 with Education Health Care (EHC) plan, in paid e
...
+		| Start Date                | Planned Duration | Total Training Price | Total Training Price Effective Date | Total Assessment Price | Total Assessment Price Effective Date | Actual Duration | Completion Status | Contract Type | Aim Sequence Number | Aim Reference | Framework Code | Pathway Code | Programme Type | Funding Line Type                                                     | SFA Contribution Percentage | Small Employer | EEFCode |
+		| 06/Aug/Last Academic Year | 12 months        | 7500                 | 06/Aug/Last Academic Year           | 0                      | 06/Aug/Last Academic Year             |                 | continuing        | Act2          | 1                   | ZPROG001      | 403            | 1            | 2              | 16-18 Apprenticeship (From May 2017) Non-Levy Contract (non-procured) | 100%                        | SEM1           | 2       |
```
Comment (ESFA-Pawan) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature]: Learner should be 19+.
Originally it was 19+, not sure when it got changed.

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature [hunk 2]
```diff
@@ -51,8 +51,8 @@ Scenario: Non-levy learner 19-24 with Education Health Care (EHC) plan, in paid
...
+		| Start Date                | Planned Duration | Total Training Price | Total Training Price Effective Date | Total Assessment Price | Total Assessment Price Effective Date | Actual Duration | Completion Status | Contract Type | Aim Sequence Number | Aim Reference | Framework Code | Pathway Code | Programme Type | Funding Line Type                                                     | SFA Contribution Percentage | Small Employer | EEFCode |
+		| 06/Aug/Last Academic Year | 12 months        | 7500                 | 06/Aug/Last Academic Year           | 0                      | 06/Aug/Last Academic Year             | 12 months       | completed         | Act2          | 1                   | ZPROG001      | 403            | 1            | 2              | 16-18 Apprenticeship (From May 2017) Non-Levy Contract (non-procured) | 100%                        | SEM1           | 2       |
```
Comment (ESFA-Pawan) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature]: Learner should be 19+.
Originally it was 19+, not sure when it got changed.

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature (hunk 1 above)
Comment (CarlHA) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature]: The learner is 19+, but due to legacy issues the funding line type is 16-18 and is used in the matching criteria.  Full details are on jira ticket PDI-31

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature (hunk 2 above)
Comment (CarlHA) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Non-Levy Learner/Small Employer/Non-Levy 19-24 learner with EHC plan with a small employer at start PV2-328.feature]: As above

**Pull request merged**